docs/Getting Started.md

```diff
@@ -0,0 +1,43 @@
+# Clone Serai
+
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+# Build and Run Serai
+
+- Ubuntu LTS or WSL2 w/ Docker (Other distributions may also work or docker can be used)
+    - Windows Guide: [https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43](https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43)

```

Comment:
Do we want to link to a personal post? I'd rather let people fend for themselves when they have poor environments than officially link to this and carry its burden.

Reply:
Will put it in the docker guide instead of here, but yes, we should link any guides that make it easier for devs to get started.